packages/sil-governance/src/routes/create-proposal/components/CommitteeSelect.tsx

```diff
@@ -0,0 +1,86 @@
+import { ElectionRule } from "@iov/bns";
+import { FormApi } from "final-form";
+import { Block, SelectFieldForm, Typography } from "medulas-react-components";
+import { Item } from "medulas-react-components/types/components/forms/SelectFieldForm";
```

Comment:
Please only import from package root `"medulas-react-components"`. Your type is exported as `SelectFieldFormItem` from the package